$OpenBSD: patch-configure,v 1.3 2016/03/18 23:57:57 tim Exp $

Fix a bug causing the default modules directory to be set incorrectly.

--- configure.orig	Sat Feb 27 13:41:16 2016
+++ configure	Fri Mar 18 20:36:09 2016
@@ -16500,7 +16500,8 @@ fi
   exec_prefix_NONE=
   test "x$prefix" = xNONE && prefix_NONE=yes && prefix=$ac_default_prefix
   test "x$exec_prefix" = xNONE && exec_prefix_NONE=yes && exec_prefix=$prefix
-  ac_define_dir=`eval echo $"${libdir}/${PACKAGE}"`
+  pkglibdir=${libdir}/${PACKAGE}
+  ac_define_dir=`eval echo $pkglibdir`
   ac_define_dir=`eval echo $ac_define_dir`
   PKGLIB_DIR="$ac_define_dir"
 
